spec/requests/authentications_spec.rb

```diff
@@ -108,17 +110,6 @@
             end.to change(User, :count).by(0)
           end
         end
-
-        it 'should not accept multiple profiles from the same source' do
```

Comment:
It was removed as regular users will not be able to do this trick anymore - it requires sending POST request instead of GET. We can cover this edge case in some lower level specs if needed.